docker-compose.yml

```diff
@@ -423,7 +423,17 @@ services:
                   -Djavax.net.ssl.trustStorePassword=confluent
                   -Djavax.net.ssl.keyStore=/etc/kafka/secrets/kafka.client.keystore.jks
                   -Djavax.net.ssl.keyStorePassword=confluent
-
+  ksql-cli:
+    image: confluentinc/cp-ksql-cli:5.0.0-beta30
+    container_name: ksql-cli
+    depends_on:
+      - kafka1
+      - connect
+      - ksql-server
+    volumes:
+      - $PWD/scripts/ksql/ksqlcommands:/tmp/ksqlcommands
+    entrypoint: /bin/sh
```

Comment:
Left the same comment on another pr, I am not sure the entrypoint is strictly needed or desirable here.